src/coprocessor/trie/mod.rs

```diff
@@ -318,17 +315,18 @@ impl<F: LurkField> CoCircuit<F> for InsertCoprocessor<F> {
 
 /// Add the `Trie`-associated functions to a `Lang` with standard bindings.
 // TODO: define standard patterns for such modularity.
-pub fn install<F: LurkField>(state: &Rc<RefCell<State>>, lang: &mut Lang<F, TrieCoproc<F>>) {
+pub fn install<F: LurkField>(state: &StateRcCell, lang: &mut Lang<F, TrieCoproc<F>>) {
     lang.add_coprocessor(".lurk.trie.new", NewCoprocessor::default());
     lang.add_coprocessor(".lurk.trie.lookup", LookupCoprocessor::default());
     lang.add_coprocessor(".lurk.trie.insert", InsertCoprocessor::default());
+    lang.intern_symbols(state);
+}
 
-    let trie_package_name: Symbol = ".lurk.trie".into();
-    let mut package = Package::new(trie_package_name.into());
-    for name in ["new", "lookup", "insert"].into_iter() {
-        package.intern(name);
-    }
-    state.borrow_mut().add_package(package);
+/// Creates a `Lang` using the `Trie`-associated functions with standard bindings
+pub fn mk_lang<F: LurkField>(state: &StateRcCell) -> Lang<F, TrieCoproc<F>> {
+    let mut lang = Lang::new();
+    install(state, &mut lang);
+    lang
```

Comment:
This is a bit of a weird public function. I would normally expect that one would want to build `Lang`s with a number of different coprocessors, composed with something like `install`. In a world where that's possible, which we target, a function like this would be little more than a convenience for testing, in most cases. What about putting this in the test module or something? I'm a little wary of adding it to the public API since that might create confusion about what is sensible.